File: api-crypto/crypto/csharp/src/IDTBSRetrieverSigner.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using tr.gov.tubitak.uekae.esya.api.common.crypto;

namespace tr.gov.tubitak.uekae.esya.api.crypto.src
{
    public interface IDTBSRetrieverSigner : BaseSigner
    {
        byte[] getDtbs();
    }
}

File: api-crypto/crypto/src/tr/gov/tubitak/uekae/esya/api/crypto/IDTBSRetrieverSigner.java
package tr.gov.tubitak.uekae.esya.api.crypto;

import tr.gov.tubitak.uekae.esya.api.common.crypto.BaseSigner;
import tr.gov.tubitak.uekae.esya.api.common.util.StringUtil;

public interface IDTBSRetrieverSigner extends BaseSigner {

    byte[] tempSignature = StringUtil.hexToByte("00000000");

    static byte[] getTempSignatureBytes() {
        return tempSignature;
    }

    byte[] getDtbs();
}
